src/main/java/gui/misc/BlurHelper.java

```diff
@@ -124,4 +142,24 @@ public void processBlurUsingBounds() {
         imageView.setImage(writableImage);
     }
 
+    /**
+     * Bind this blurhelper to scrollchanges in a watchable scrollpane.
+     * @param watchable the scrollpane to watch.
+     */
+    public void watchScrolling(ScrollPane watchable) {
+        ChangeListener<Number> listener = new ChangeListener<Number>() {
```

Comment:
maybe extract this into a seperate method and use a lambda, also use a more descriptive name